discord/ext/pages/pagination.py

```diff
@@ -254,6 +254,8 @@ class PageGroup:
         Whether the buttons get disabled when the paginator view times out.
     use_default_buttons: :class:`bool`
         Whether to use the default buttons (i.e. ``first``, ``prev``, ``page_indicator``, ``next``, ``last``)
+    use_styled_buttons: :class:`bool`
+        The same as ``use_default_buttons`` but uses emojis for labels, overwrites ``use_default_buttons`` when used.
```

Comment:
Please add a versionadded for v2.4

Reply:
Here's an example from another open PR https://github.com/Pycord-Development/pycord/pull/1809/files#diff-f10a1b29b4a4c6c39210d6566a9c849b92f0e7e35746c78c61be6ac8fcb5c417R184-R189

Reply:
Ok. should I have it as added in 2.4?